docs/source/devguide.rst

```diff
@@ -21,11 +18,14 @@ Developer's Guide
   Custom libraries
   ****************************************
 
-****************
-Our contributors
-****************
+*****************
+Our collaborators
+*****************
+
+The following collaborators are ordered alphabetically:
 
-   1. Matt Richardson - `Github Account <https://github.com/mattallen37/>`__
-   2. Nicole Parrot - `Github Account <https://github.com/cleoqc/>`__
-   3. Robert Lucian Chiriac - `Github Account <https://github.com/RobertLucian/>`__
-   4. John Cole - `Github Account <https://github.com/johnisanerd/>`__
+   * John Cole - `Github Account <https://github.com/johnisanerd/>`__.
+   * Matt Richardson - `Github Account <https://github.com/mattallen37/>`__.
+   * Nicole Parrot - `Github Account <https://github.com/cleoqc/>`__.
+   * Robert Lucian Chiriac - `Github Account <https://github.com/RobertLucian/>`__.
+   * Shoban Narayan - `Github account <https://github.com/shoban94>`__.
```

Comment:
Karan is missing

Reply:
I couldn't see Karan as a contributor to this repository nor to the development of the GoPiGo3's hardware.

Reply:
sorry, I thought I was in the DI Sensors repo.